Extensions/hideavatars.js

```diff
@@ -34,11 +34,25 @@ XKit.extensions.hideavatars = new Object({
g31pYqta6wABIQQorcyzOxS3JBMFWOLGVom8Big970UDes+pnW833qedORt/pE93NyzK1ZwHWjxlEBMoo/0v+uq6lEBGyjJBRBsrYEZJvgWFFqKgHUBUT5CYHEIoWszph65N7cPZlpwIALrnpXVhyx9+R1KIgpDmJiVyPGWfOm4Lead15lEljrHtki35ewIzTz5uCU2b2AhGjMZph/fItiKJmIdKvCBU1QTCCZIjaKGk5n4ix+v6tRgCnvPsdOPGMbhzcNVjB5qCAEuM/63Zj0bLrEOVyw+AbQ9j42EvOOICRdsS4/s4Pmxl+e8eTQCQPQ5TcQXG5mBVUl74XABSzgqol5hOeW/Ff89ulX5yFRr1h5zE+7LmcCGWIgRf+Yuf4yC1zDIaUrtJoEC765GwzZuC1QezYsNvhKA53gF4RKjCgGBR5Gyo36nEBLX2WFvHX31gsmDJzEqa8s8cUJSW26HX6/O/SxWsxNtww1+d/erbFAwYm9nZivhDA/V9ZiSjWcUIqUVR9DYrFMsQ4JgL3wSVQmYcIYjkhSoD1D203v1/2pXPRGGt4bI5CEBSC7hifYtWDm8z1iz8zG+m4BACj0chw+U3vNdd2/msfho6MKowxVKLvAmSjgOeR5NFMjwJbHqAACxjrH95mfj5hRjdOndXragTwXM7dPDHhmaVb8cbeQesKN1+ARiPDSf2TcM6l/bngMsYDt68EIi86OCGQPJ7grhdrByM+Kalic0HoLBOMWoyn7v+33fyXz0OjntmNmUKHxgOK8NQRYfnda82Y8z86E53d47Dg1nnmt/XLN6PWmQjlkBoCuZIJFiBIfjJUYU6OT7HC5kR43PDHF82YyVMnYvp7+jyq6+KHBl6vvLAfOzftM+Nu+cU1mD77JADA0YERrLx3bchP2CNssPwgdPHCAuDnAk21rjCropogY3QyLsaffrzBXL/ittyE7b3eemDnsISZESXAI99fZcZNnNxlvq+8dy3SjjjUulId8oG3FBJyC8ilRcEA6+tS6ySzQ08ELnMkvPDULjTqGQBg0pQJmHH+ScV8fhTwrY3MuOGjo3j20W3O+H0vHcLzq3Z4yiHnL3mpNClFUTJhUDQVQIYsVqio2KSbQXqRA4w4BVYsfsaMWfDV96FRz5RkiNW6Q2ll2//xqjN+1+Z9SNJIpbtauOagLO64AClMyYv9fmZWkdGRKJuV921/ZjdGj9ZzE+7rwplzTw7XY2W94r/hoyNYcOsH3JT76lmFO7m+Xo1ZfhiUFSGVm7uSJNbDoh8F3Kwv5w1RGmHJXX8z465ZdCEajUZFPUBWovJ5Lv3s+eg5YXyQdt9411WgjJz6no9Z5LmFtl7s1/UdVFa0bgTit5QwgvJU+X3npn0YPJSny109HThr3lTVBXwmNzI0hssXzjXjlt+zxnw/e95p6DlxvPB1H7NcSyAv7pbPEpNCbV1flxO7ZgbvHvLreiU7jIFHvmfR/NpvzEeW+etRUPv7+Dc/hLSWAABe3rQHq3+3EVvW7TT3LfzB1UWuwXo49KxDA90YSkgLCowgdWJoVSR2uUJ5757tBzHw2lEAQMf4GvqmdasWUFLjpBZh7pVnO9rvnFjDsp8+bX47ub8PZ82Z5vETuVfXncPIQ4iJNa5MajXWMjhSyYyvBVlIQcR46NtPmPEdXbXgXgOmGeFzd15hrm9ZtxO7d+wHg3Fo75tYu8zmCTd85yqMDI+GxZpC6+T1E/l7jUMeCFNY1M7lbZmMfAhwipJaZfnAqwN4/ZU39a4tgS190ybhjPNONddX/HwN0nE55Y2TCH/+9TpzbcKkTlx2/VxH62TyDXKKs2E6LKOAl5yARXhDWRj1zwmgnLdZ6/EPLyMAD37rsVAA4jC2PlrHwh9eaa5teHwbXn9lwLGmocERrLzPAuKCL1yC0ZExU8AFy0qxYuHicCXW4nr+kJlzHGabNN3au3bYEAInmXL64YOD2PviQaUekPvkrIv7MfnkHnPt0QfWIk4jp9iJCFizdKMZk9YS3Pjdq00Bl70CSshbbJhO1Q4KJjCSJm0s+WZe+uce7N/ZhTiJMHR41D0UVY+xAETALxetxLWLPmisLmvk7DDLMsyaPwObVu8wjG/gwGHESRycLNfHGPd9fQkuXHAOACBOYmRZhjiJKnKCsN7hNEiAw7DU/AwOeOJXG9voAgv7+8ZG6nj4rieCuaM4wu/vfspZK0oi9WCWwdj27MvY+uzL5kGjBLa7zTsfJDUZgm2Q4ApicjzNDnpjk9/1UX3c1c4RutNuLztZ1G4yvRJtGiSCPjpWTL9JS31ln1Bb7Wytev+UnkP1xFhrveOgWcNXcKqf1VEbPXjNz+OBsNfH7w6raoFr1vPTTi+SGrrVZA8lCEKRkNtgJN+yaOXr1d1dVQ2XzdtsXSbXXv+Sv27QIGE7RCxTc4iQQyFb9flWdG+2aIHzX61p1gXm9gtWtcVW9zFKFye3W1xpljYHjWgb4f1QiYpGqupmqKr+4iqQU4TbrAkrsACSFqC4AB9DE6PTBaY0MWr4cEy9f66JH2tzddglVnaPCxCUIOHk2C3cQO/zpaAltUrrlS9MeZGl/R4hPVK5/QxWibZJyi9vtYytpMR02zyjIvxx9/miSQtcBQeRwkVYEhPd4hS2yLTb31cZ2trVelXHZ1VXOVr7eoBZFS5e3JXK5MUvi7endb+1rXV/Ybvd3a2Fzmo41PsUPRdwQdC1gKHDw0U22I7WdSZ3vL5uHogksFLYRuOBHLjZuowjA0d1Jmh8XVjAT5677f/8tTkIDFDS4bfFm4NOuzwIuza/hiU/Wu35j+gd8hORKHz91U974YBicYJYvJYLh9Cwf7ogXtmtms9/jZbD+cprkXRTYNvzL8pu8Rzw3th3BIf2Hm5BNDh4N7eVrzd9e7siyjR9ebribbHqXITg5jtuBppWVXzQMleXdJfaoLHNs7aW+Qa32zrvpvetmOf/ALRjiTXiJEtlAAAAAElFTkSuQmCC");
```

Comment:
Instead of listing the whole thing thrice, can we store `data:image/png;base64,...` as a variable someplace and refer to it?